demosplan/DemosPlanCoreBundle/DataFixtures/ORM/ProdData/LoadUserData.php

```diff
@@ -118,8 +116,6 @@ public function createSuperUser(OrgaType $orgaTypeOlauth, ObjectManager $manager
     }
 
     /**
-     * @throws DBALException
-     * @throws ORMException
      * @throws OptimisticLockException
```

Comment:
OptimisticLockException can also be removed isn't it?

Reply:
done in [685364f](https://github.com/demos-europe/demosplan-core/pull/2960/commits/685364f97784d85dd520137c6bcee9f67c93e07c)